WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewModelTests.swift

```diff
@@ -2,9 +2,80 @@ import XCTest
 import TestKit
 @testable import Yosemite
 @testable import WooCommerce
+import Networking
```

Comment:
This import seems that do not belong here :D 
```suggestion
```

Reply:
Hmm, I thought it was needed but perhaps not any more. Thanks.